Documentation/torcx.md

```diff
@@ -0,0 +1,67 @@
+# Torcx configuration managed by Tectonic
+
+This document gives a brief overview on all torcx details managed by Tectonic.
+
+# Torcx profile
+
+All Tectonic nodes will boot into a torcx profile named `tectonic`. This can be verified via:
+
+```
+$ cat /etc/torcx/next-profile
+
+tectonic
+```
+
+This profile will pin down the container runtime version to be used by the kubelet. It can be verified via:
+
+```
+$ jq . /etc/torcx/profiles/tectonic.json
+
+{
+  "kind": "profile-manifest-v0",
+  "value": {
+    "images": [
+      {
+        "name": "docker",
+        "reference": "1.12"
+      }
+    ]
+  }
+}
+```
+
+The result of the example above is that the `docker:1.12` addon will be applied on the next machine boot.
```

Comment:
add-on

Reply:
We consistently use "addon" everywhere else:
 * https://github.com/coreos/torcx
 * https://github.com/coreos/docs/blob/master/torcx/what-is-torcx.md